integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1295,7 +1295,9 @@ protected void initialize(Map<String, Object> inputDomainMap) throws Exception {
         "cp -rf " + BaseTest.getProjectRoot() + "/kubernetes/samples " + BaseTest.getResultDir());
 
     this.voyager =
-        System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER");
+        (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
+            || (inputDomainMap.containsValue("VOYAGER"));
```

Comment:
change `inputDomainMap.containsValue("VOYAGER")` to `inputDomainMap.containsKey("LB_TYPE") && ((String)inputDomainMap.get("LB_TYPE")).equalsIgnoreCase("VOYAGER")` . so that it will be clear for someone to understand which attribute value is being checked. 
Also if you are done with changes, you can update the PR with jenkins results.